### DISCUSSION TITLE
[BUG]当1panel数据盘配置成nfs远程目录的时候，面板响应速度会变得非常慢，甚至超时

### DISCUSSION THREAD

**Discussion by wwtao1024** [1Panel-dev/1Panel#6027] [Q&A]
### 1Panel 版本

1.10.13

### 问题描述

当1panel数据盘配置成nfs远程目录的时候，面板响应速度会变得非常慢，甚至超时

### 重现步骤

当1panel数据盘配置成nfs远程目录的时候，面板响应速度会变得非常慢，甚至超时

### 附加信息

![image](https://github.com/user-attachments/assets/6e6f2234-124c-4885-a0b3-8c395826f087)

**Comment by wanghe-fit2cloud** [comment]
建议将 1Panel 的数据目录配置在本地存储上，这样可以避免网络延迟，提高数据读写的效率。

**Reply by wwtao1024** [reply]
那也不能这么离谱啊 都是内网的东西 这么说k8s是不是也不能用nfs协议的存储给pod提供持久化存储

**Comment by wwtao1024** [comment]
![image](https://github.com/user-attachments/assets/53fcb247-9bcb-4d80-a378-18b9ae8aea0d)
